apps/dotcom/client/src/tla/components/TlaIcon/TlaIcon.tsx

```diff
@@ -26,14 +28,20 @@ export function TlaIcon({
 		ref.current.style.webkitMask = getMaskStyle(icon)
 	}, [ref, icon])
 
+	const _className = classNames({
+		[styles.icon]: true,
+		[styles.inline]: inline,
+		[className]: true,
+	})
+
 	if (icon === 'none') {
-		return <div className={classNames(styles.icon, className)} />
+		return <span className={_className} />
```

Comment:
had to change the icons to spans because otherwise they can't be used as children of paragraph elements.